utilities/boostHistHelpers.py

```diff
@@ -481,3 +481,47 @@ def set_flow(h, val="nearest"):
             nearest_vals = np.take(h.values(flow=True), -2, i) 
             np.take(h.values(flow=True), -1, i)[...] = nearest_vals if val == "nearest" else np.full_like(nearest_vals, val)
     return h
+
+def swap_histogram_bins(histo, axis1, axis1_bin1, axis1_bin2, axis2=None, axis2_slize=None, flow=False, axis1_replace=None):
```

Comment:
Can you really envision a use case where this the two axes being swapped are different? You would need some shape consistency which would be a pain

Reply:
where which two axes are different?

Reply:
Ok I think I'm confused about what axis1 and axis2 are

Reply:
axis1_bin1 is swapped with axis1_bin2, both bins belong to axis1. The swapping is done for all bins in axis2_slize which is a set of bins in axis2